src/librekey/rnp_key_store.cpp

```diff
@@ -569,13 +569,18 @@ rnp_key_store_import_key(rnp_key_store_t *        keyring,
     }
     exkey = rnp_key_store_get_key_by_grip(keyring, pgp_key_get_grip(srckey));
     expackets = exkey ? pgp_key_get_rawpacket_count(exkey) : 0;
+    keyring->disable_validation = true;
     if (!(exkey = rnp_key_store_add_key(keyring, &keycp))) {
         RNP_LOG("failed to add key to the keyring");
         pgp_key_free_data(&keycp);
         return NULL;
```

Comment:
This leaves validation disabled on the keyring

Reply:
@dewyatt Thanks, fixed this.